procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -22,13 +22,17 @@
     "import time\n",
```

Comment:
<p>Why did you move this down?</p><p> </p><p>It was up top because it helps diagnose issues... in fact does this even work? I don't think you've enabled anything?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>What do you think If I move back the checks where they were, and leave the section "Run prepare for flats" at the end of the notebook. This way calibrations can start immediately after that. </p>

Reply:
<p>I guess I'm not understanding why you want to change it. Is this to save time?</p><p>The reason prepare for flats runs first is because it tests out the mirror covers and pneumatics etc. </p><p>It also moves in and out of the dome following.</p><p>In the future this will be a script, or more likely a series of scripts, might be best to just leave it.</p>

Reply:
<p>Done</p>